tests/drink/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+mod psp22_contract;
+mod test_contract;
+
+#[cfg(test)]
+mod psp22_tests;
+#[cfg(test)]
+mod test_contract_tests;
+
+use drink::AccountId32;
+
+pub const ALICE: drink::AccountId32 = AccountId32::new([2u8; 32]);
+pub const BOB: drink::AccountId32 = AccountId32::new([1u8; 32]);
```

Comment:
in case of `MinimalRuntime`, `BOB` is rich sudo - are you sure that you want to be implicit about it?